python/ray/train/_internal/state/schema.py

```diff
@@ -53,7 +53,6 @@ class TrainWorkerInfo(BaseModel):
     resources: Dict[str, float] = Field(
         description="The resources allocated to the worker."
     )
-    worker_log_file_path: str = Field(description="The path to the worker log file.")
```

Comment:
Just to confirm: Schema removals here are okay as long as we make sure to no longer depend on this in the front-end.

Reply:
Yeah this schema isn't consumed anymore